box/stories/index.stories.tsx

```diff
@@ -1,15 +1,23 @@
 import React from 'react';
-import { Box } from "../index";
-import './box-stories.scss';
+import { Box } from '../index';
+import './box-stories.css';
```

Comment:
@m4dz what have you done ? I have spend some time to fix all of these on dockit react.
Why we do have again those changes ?